live/Makefile

```diff
@@ -80,4 +89,4 @@ shellcheck:
 	@find config-cdroot root src -type f -exec grep -l -E "^#! *(/usr/|)/bin/(ba|)sh" \{\} \; \
```

Comment:
Not SLES related, but since we're touching this Makefile:
`make shellcheck` is looking for `root` which is no longer here